libraries/anvill_passes/src/JumpTableAnalysis.cpp

```diff
@@ -0,0 +1,639 @@
+#include <anvill/ABI.h>
```

Comment:
Can you add some more comments into this code that describes what / why it's doing things, and perhaps have a file-level comment that describes the approach at a high-level?